### PR TITLE
docs(api) dynamic expressions in import()

### LINKED ISSUE #1699: API - Delineate and Elaborate on Dynamic Expressions
I searched and searched but couldn't find any mention of this feature in the new website.

https://webpack.github.io/docs/context.html

It speaks of *dynamic imports*:
```js
require("./template/" + name + ".jade");
```

So I was wondering first of all - is that on purpose? ie. is this feature deprecated? If not, and if it really isn't in the docs, then I can start a PR for updating the docs with this useful information.

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Fixes #1699

### CODE REVIEW

File: src/content/api/module-methods.md [hunk 1]
```diff
@@ -58,12 +59,13 @@ export default {
 
 ### `import()`
 
-`import('path/to/module') -> Promise`
+`function(String path):Promise`
```
Comment (montogeek) [inline src/content/api/module-methods.md]: Let's stick with `string` https://github.com/webpack/webpack.js.org/search?q=string&unscoped_q=string

File: src/content/api/module-methods.md [hunk 2]
```diff
@@ -74,6 +76,24 @@ if ( module.hot ) {
...
+
+```javascript
+// imagine we had a method to get language from cookies or other storage
+const language = detectVisitorLanguage();
+import(`./locale/${language}.json`).then(module => {
+  // do something with the translations
+});
+```
+
+T> Using the [`webpackInclude` and `webpackExclude`](/api/module-methods/#magic-comments) options allows us to add regex patterns that reduce the files that webpack will bundle for this import.
```
Comment (montogeek) [inline src/content/api/module-methods.md]: > allow you

or

> allow

File: src/content/guides/code-splitting.md [hunk 3]
```diff
@@ -280,6 +280,8 @@ __src/index.js__
   });
 ```
 
+T> It is possible to provide a [dynamic expression](/api/module-methods/#dynamic-expressions-in-import) to `import()` when you might need to import specific module based on a computed variable later.
```
Comment (montogeek) [inline src/content/guides/code-splitting.md]: Last part sounds weird, What you think about:
> based on a variable computed later.

File: src/content/api/module-methods.md (hunk 1 above)
Comment (EugeneHlushko) [reply src/content/api/module-methods.md]: my god where did i grab that Camel case :)

File: src/content/guides/code-splitting.md (hunk 3 above)
Comment (EugeneHlushko) [reply src/content/guides/code-splitting.md]: we are trying to say: "use `import()` to import something later (maybe based on a variable that you will compute)"

File: src/content/api/module-methods.md (hunk 1 above)
Comment (EugeneHlushko) [reply src/content/api/module-methods.md]: updated

File: src/content/api/module-methods.md (hunk 2 above)
Comment (EugeneHlushko) [reply src/content/api/module-methods.md]: changed to `allows you` and also improved the next statement

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**